en/pd-recover.md

```diff
@@ -0,0 +1,197 @@
+    ```
+
+    In the command above, `${version}` is the version of the TiDB cluster, such as `v4.0.0-rc`.
+
+2. Unpack the TiDB package for installation:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tar -xzf tidb-${version}-linux-amd64.tar.gz
+    ```
+
+    `pd-recover` is in the `tidb-${version}-linux-amd64/bin` directory.
+
+## Recover the PD cluster
+
+This section introduces how to recover the PD cluster using PD Recover.
+
+### Get Cluster ID
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl get tc ${cluster_name} -n ${namespace} -o='go-template={{.status.clusterID}}{{"\n"}}'
+```
+
+Example:
+
+```
+kubectl get tc test -n test -o='go-template={{.status.clusterID}}{{"\n"}}'
+6821434242797747735
+```
+
+### Get Alloc ID
+
+When you use `pd-recover` to recover the PD cluster, you need to specify `alloc-id` which is larger than the largest allocated ID (`Alloc ID`) of the original cluster.
+
+1. Access the Prometheus monitoring data of the TiDB cluster by taking steps in [Access the monitoring data](monitor-a-tidb-cluster.md#access-the-monitoring-data).
+
+2. Enter `pd_cluster_id` in the input box and click the `Execute` button to make a query. Get the largest value in query result.
+
+3. Multiply the largest value in the query result by `100`. This multiplied value is the `alloc-id` value specified when using `pd-recover`.
+
+### Recover the PD Pod
+
+1. Delete the Pod of the PD cluster.
+
+    Execute the following command to set the value of `spec.pd.replicas` to `0`:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl edit tc ${cluster_name} -n ${namespace}
+    ```
+
+    Because the PD cluster is in an abnormal state, TiDB Operator cannot synchronize the change above to PD StatefulSet. You need to execute the following command to set the value of PD StatefulSet `spec.replicas` to `0`.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl edit sts ${cluster_name}-pd -n ${namespace}
+    ```
+
+    Execute the following command to confirm that the PD Pod is deleted:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl get pod -n ${namespace}
+    ```
+
+2. After confirming that all PD Pods are deleted, execute the following command to delete the PVC bound to the PD Pods:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl delete pvc -l app.kubernetes.io/component=pd,app.kubernetes.io/instance=${cluster_name} -n ${namespace}
+    ```
+
+3. After the PVC is deleted, increase the capacity of the PD cluster to one Pod:
+
+    Execute the following command to set the value of `spec.pd.replicas` to `1`:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl edit tc ${cluster_name} -n ${namespace}
+    ```
+
+    Because the PD cluster is in an abnormal state, TiDB Operator cannot synchronize the change above to PD StatefulSet. You need to execute the following command to set the value of PD StatefulSet `spec.replicas` to `1`.
```

Comment:
```suggestion
    Because the PD cluster is in an abnormal state, TiDB Operator cannot synchronize the change above to the PD StatefulSet. You need to execute the following command to set the `spec.replicas` of the PD StatefulSet to `1`.
```